diff --git a/containers/flat_tree.cc b/containers/flat_tree.cc
index bda3855d0..43fd7bcca 100644
--- a/containers/flat_tree.cc
+++ b/containers/flat_tree.cc
@@ -8,7 +8,7 @@
 // on build time. Try not to raise this limit unless absolutely necessary. See
 // https://chromium.googlesource.com/chromium/src/+/HEAD/docs/wmax_tokens.md
 #ifndef NACL_TC_REV
-#pragma clang max_tokens_here 260000
+#pragma clang max_tokens_here 350000
 #endif
 
 namespace base {
