diff --git a/config/linux/BUILD.gn b/config/linux/BUILD.gn
index 47704248b..0820ae643 100644
--- a/config/linux/BUILD.gn
+++ b/config/linux/BUILD.gn
@@ -15,6 +15,21 @@ group("linux") {
 # is applied to all targets. It is here to separate out the logic that is
 # Linux-only. This is not applied to Android, but is applied to ChromeOS.
 config("compiler") {
+  if(is_linux) {
+    cflags_cc = [
+      "-nostdinc++",
+    ]
+    ldflags = [
+      "-lc++",
+      "-lc++abi",
+    ]
+    lib_dirs = [
+      "../../../../../libcxx/lib",
+    ]
+    include_dirs = [
+      "../../../../../libcxx/include/c++/v1",
+    ]
+  }
 }

 # This is included by reference in the //build/config/compiler:runtime_library
